<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XMLDialog</class>
 <widget class="QDialog" name="XMLDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XML Editor</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="apoXmlTreeView" name="xmlTree" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>351</width>
     <height>611</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="tabKeyNavigation" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragEnabled" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonOK">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>600</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>600</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="apoUiCommonXmlView" name="xmlTable" native="true">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>521</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>apoXmlTreeView</class>
   <extends>QWidget</extends>
   <header>apoXmlTreeView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>apoUiCommonXmlView</class>
   <extends>QWidget</extends>
   <header>apoUiDetailView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
